src/metabase/driver/mongo/util.clj

```diff
@@ -1,6 +1,7 @@
 (ns metabase.driver.mongo.util
   "`*mongo-connection*`, `with-mongo-connection`, and other functions shared between several Mongo driver namespaces."
   (:require [clojure.tools.logging :as log]
+            [metabase.util.ssh :as ssh]
```

Comment:
I've been trying to keep all the requires ordered, otherwise they can get out of hand

The rule is

1.  All Clojure requires, sorted alphabetically
2.  All 3rd-party requires, sorted alphabetically
3.  All Metabase project requires, sorted alphabetically

So this should probably go close to the bottom of the list